src/options/options.ts

```diff
@@ -7,9 +7,9 @@ export interface TsAutoMockOptions {
   cacheBetweenTests: TsAutoMockCacheOptions;
 }
 
-let options: TsAutoMockOptions = null;
+let options: TsAutoMockOptions | undefined;
```

Comment:
In reality, the options passed to SetTsAutoMockOptions are a partial of TsAutoMockOptions. To avoid spreading undefined  we could do something like this.

```ts
const options: TsAutoMockOptions = defaultOptions;

export function SetTsAutoMockOptions(_options: Partial<TsAutoMockOptions>): void {
  Object.keys(_options).forEach((key: string ) => {
    options[key] = _options[key];
  });
}
export function GetOptionByKey<T extends keyof TsAutoMockOptions>(optionKey: T): TsAutoMockOptions[T] {
  return options[optionKey];
}
```